packages/cli/test/e2e/voluntaryExitRemoteSigner.test.ts

```diff
@@ -0,0 +1,171 @@
+import path from "node:path";
+import fs from "node:fs";
+import {GenericContainer, Wait, StartedTestContainer} from "testcontainers";
+import tmp from "tmp";
+import {retry} from "@lodestar/utils";
+import {ApiError, getClient} from "@lodestar/api";
+import {config} from "@lodestar/config/default";
+import {interopSecretKey} from "@lodestar/state-transition";
+import {spawnCliCommand, execCliCommand} from "@lodestar/test-utils";
+import {getMochaContext} from "@lodestar/test-utils/mocha";
+import {testFilesDir} from "../utils.js";
+
+let web3signerUrl: string;
+// using the latest image to be alerted in case there is a breaking change
+const web3signerVersion = "latest";
+
+describe("voluntaryExit using remote signer", function () {
+  const testContext = getMochaContext(this);
+  this.timeout("60s");
+
+  let startedContainer: StartedTestContainer;
+
+  after("stop web3signer container", async function () {
+    await startedContainer.stop();
+  });
+
+  before("start web3signer container", async function () {
+    this.timeout("300s");
+    // path to store configuration
+    const tmpDir = tmp.dirSync({
+      unsafeCleanup: true,
+      // In Github runner NodeJS process probably runs as root, so web3signer doesn't have permissions to read config dir
+      mode: 755,
+    });
+    // Apply permissions again to hopefully make Github runner happy >.<
+    fs.chmodSync(tmpDir.name, 0o755);
+
+    const configDirPathHost = tmpDir.name;
+    const configDirPathContainer = "/var/web3signer/config";
+
+    // keystore content and file paths
+    const passwordFilename = "password.txt";
+    const password = "password";
+
+    const keystoreStrings = getKeystoresToExit();
+    for (const [idx, keystoreString] of keystoreStrings.entries()) {
+      fs.writeFileSync(path.join(configDirPathHost, `keystore-${idx}.json`), keystoreString);
+    }
+    fs.writeFileSync(path.join(configDirPathHost, passwordFilename), password);
+    const port = 9000;
+
+    startedContainer = await new GenericContainer(`consensys/web3signer:${web3signerVersion}`)
```

Comment:
Would be great to make the web3siger initialization reusable to de-duplicate a lot of code between this test case and the web3signer signature test

https://github.com/ChainSafe/lodestar/blob/8afbc98b561d714fdadd1946ec2a5aab83f479de/packages/validator/test/e2e/web3signer.test.ts#L26

Something like `startValidatorWithKeyManager` just for web3signer would be awesome

https://github.com/ChainSafe/lodestar/blob/8afbc98b561d714fdadd1946ec2a5aab83f479de/packages/cli/test/utils/validator.ts#L11

Reply:
Let's remove the mocked version, I don't see any use case for this at this point. Better to use web3signer container in e2e tests

Reply:
Looks great, we recently just added `test-utils` package but that's exactly the use case for it.